faust/utils/cron.py

```diff
@@ -0,0 +1,14 @@
+import datetime
+import time
+from croniter.croniter import croniter
+
+
+def secs_for_next(cron_format: str, tz: any = None) -> float:
+    now_ts = time.time()
+    # If we have a tz object we'll make now timezone aware, and
+    # if not will set now to be the current timestamp (tz
+    # unaware)
+    # If we have tz, now will be a datetime, if not an integer
+    now = tz and datetime.datetime.now(tz) or now_ts
```

Comment:
@ask This is a bit hacky but the other solution was to add another dependency to figure out the local timezone. 
Ideally we would have something like 
```python
def secs_for_next(cron_format: str, tz: any = None) -> float:
    timezone = tz or localtimezone
    now_datetime = datetime.datetime.now(timezone)
    cron_it = croniter(cron_format, start_time=now_datetime)
    return cron_it.get_next(float) -  time.time()
```

Reply:
Note that I've isolated this function in utils so that we could easily test it.